cypress/integration/GlossaryAppIntegration/GlossaryAppIntegration.js

```diff
@@ -0,0 +1,152 @@
+  cy.get('div.radio-selection label').last().should('include.text', contains);
+});
+
+Then('{string} is selected by default', (a) => {
+  cy.get('div.radio-selection').find("input[value='starts with']").should('be.checked')
+});
+
+Then('{string} option is displayed below search bar', (browse) => {
+  cy.document().then(doc => {
+    const siblingEl = doc.querySelector('span.browse');
+    expect(siblingEl.firstChild.wholeText).to.include(browse);
+    expect(siblingEl.parentNode.previousSibling.className).to.be.eq('dictionary-search');
+  })
+});
+
+Then('browse option contains all alphabetic letters and {string} appears at the end of the list', (a) => {
+  cy.document().then(doc => {
+    const browseOptions = doc.querySelectorAll('nav.az-list ul li');
+    for (let i = 0; i < browseOptions.length - 1; i++) {
+      let int = browseOptions[i].textContent;
+      expect(int).to.match(/[A-Z]/);
+    }
+    expect(browseOptions[browseOptions.length - 1].textContent).to.be.eq(a)
+  });
+
+});
+
+Then('each option appears as a link', () => {
+  cy.get('nav.az-list ul li a').each((el) => {
+    expect(el).to.have.attr('href');
+  })
+});
+
+Then('the page is showing the expand results for letter {string}', (letter) => {
+  let results;
+  
+  cy.url().then(url => {
+    if (url.includes('espanol'))
+      results = `resultados de: ${letter}`;
+    else
+      results = `results found for: ${letter}`;
+
+    cy.get("div[class='dictionary-list-container results'] h4").should('include.text', results);
+  });
+});
+
+Then('search bar is displayed below introductory text', () => {
+  cy.document().then(doc => {
+    expect(doc.querySelector('form.search-box-container').previousElementSibling.firstChild.tagName).to.be.eq('P');
+  });
+});
+
+Then('{string} is displayed as the submit button', (searchButton) => {
+  cy.get('input#btnSearch').should('have.value', searchButton);
+});
+
+Then('user is on landing page for the Genetics Dictionary', () => {
+  cy.location('pathname').should('be.eq', '/publications/dictionaries/genetics-dictionary');
+});
+
+Then('the following introductory text appears below the page title: {string} and {string}', (beforeCount, afterCount) => {
+  cy.document().then(doc => {
+    const paragraph = doc.querySelector('#NCI-glossary-root div p').textContent;
+    expect(paragraph).to.include(afterCount);
+    expect(paragraph).to.include(beforeCount);
+  });
+});
+
+Then('there is a dynamic number generated by the API that has been inserted', () => {
+  cy.get('#NCI-glossary-root div p').should('not.include.text', '{{term_count}}');
+  cy.document().then(doc => {
+    const paragraph = doc.querySelector('#NCI-glossary-root div p').textContent;
+    expect(paragraph).to.match(/^(?=.*\d).+$/);
+  });
+});
+
+Then('{string} links to {string}', (link, href) => {
+  cy.get(`a[href='${href}']`).should('include.text', link);
+});
+
+Then('user is redirected to 404 error page', () => {
```

Comment:
This is probably not needed. There is no redirect to a 404 (a redirect is a HTTP thing and using the term here is confusing). 
```gherkin
Given a user visits the url "/foo/bar"
Then the status is 404
   And the content title is "Page not found"
```
Or something like that.